packages/plugins/src/SidebarPlugin.ts

```diff
@@ -205,6 +205,12 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
             if (currPage !== undefined) {
               createNavigationRefs(currPage, prevPage, nextPage);
             }
+            if (currPage === undefined) {
+              console.warn(
+                '\x1b[31m',
+                `*** PAGE RETURNING UNDEFINED -  PLEASE CHECK ALL YOUR DIR'S CONTAIN INDEX PAGES. THIS MAY CAUSE ISSUES WITH SIDEBAR AND NAVIGATION ***`
```

Comment:
console.warn - coloured in red and with *** to make it stand out. No index pages will cause pages with sidebar to fail so wanted to be very visible in the log

Reply:
capitals is rather shouty..

not having an index is a valid, if we want hidden page support

who discovers the error, the user or the site maintainer ?
If the build fails, the site maintainer does
If the page fails the user does

Worth a discussion on the problem statement to understand what solution is best

formatted logs don’t work so well in splunk 

Might be worth considering this in a wider context, how should we handle page errors in general ?

Next has a red box and overlay to report page errors. Typically markdown solutions never fail and always try to display content.